micronaut-maven-integration-tests/src/it/dockerfile-docker-native-distroless/Dockerfile

```diff
@@ -3,6 +3,8 @@ WORKDIR /home/app
 
 COPY classes /home/app/classes
 COPY dependency/* /home/app/libs/
+COPY graalvm-reachability-metadat[a] /home/app/graalvm-reachability-metadata
```

Comment:
Why the brackets `[a]` and `[e]`?

Reply:
It's a trick to make the Docker build not to fail if the directories don't exist 🤓

Reply:
TIL, thanks!